### ISSUE TITLE
TypeError on Blender 3.1 beta

### ISSUE THREAD

**Issue by isbm** [EatTheFuture/camera_shakify#6]
Crashes. Wants `int`, gets `float`:

<img width="934" alt="Screenshot 2022-02-01 at 01 45 21" src="https://user-images.githubusercontent.com/1073714/151896681-97e8e97d-c82d-4684-96f9-10ea6a17fcb0.png">

**Comment by cessen** [comment]
Thanks for the report!

What build of Blender are you using?  3.1 isn't out yet (latest release is 3.0.1), so I presume you're using a nightly build?

**Comment by isbm** [comment]
Well, what else to use that crashes less elsewhere? :wink:  I just filled out other, more serious bug for 3.1, when it coredumps me on just a simple video on background. Do you assume this is a bug in 3.1 and the type will be again back to `int` at some point, or it still makes sense to keep it all up to date?

**Comment by cessen** [comment]
I don't know if this is a bug with Blender or Camera Shakify.  But if I know what build you are using, it will help me figure it out.

Usually when you download a non-release build, it has a commit hash in the file name.  For example, the latest 3.2 alpha build for linux has this filename:

```
blender-3.2.0-alpha+master.95005bbe02cf-linux.x86_64-release.tar.xz
```

The `95005bbe02cf` is the commit hash.

Can you please tell me the commit hash of the build you are using?

**Comment by SuperDepressing** [comment]
I have the same issue with 
`blender-3.1.0-beta+v31.f420118335e4-windows.amd64-release.zip`

**Comment by cessen** [comment]
Thanks guys!

> But I don't think this is a problem with a particular commit

Agreed, but I wanted to make sure I wouldn't be testing on a too-early commit, possibly before the relevant change was made in Blender.

**Comment by marianobasti** [comment]
Line 209 and 210 in init, cast action_range to int, such as
`    constraint.frame_start = int(action_range[0])
    constraint.frame_end = int(action_range[1])`

fixed

**Comment by isbm** [comment]
> Line 209 and 210 in init, cast action_range to int, such as `constraint.frame_start = int(action_range[0]) constraint.frame_end = int(action_range[1])`
> 
> fixed, works even in 3.2

That's more like a workaround to me. I wondering why `action_range` contains floats and before it was OK. Some better refactoring is needed, IMO.

**Comment by marianobasti** [comment]
I figured it was something internal about blender. Doesn't seem like
something critical tho.

**Comment by cessen** [comment]
Thanks for your help, guys!  Fixed in commit 3088d52e1090bd71a5d04f110663204e372875b5.
